pkg/grpc/tracing/tracing.go

```diff
@@ -59,6 +60,14 @@ func GetProcessKprobe(event *MsgGenericKprobeUnix) *tetragon.ProcessKprobe {
 			Pid:       &wrapperspb.UInt32Value{Value: event.ProcessKey.Pid},
 			StartTime: ktime.ToProto(event.ProcessKey.Ktime),
 		}
+		go func() {
+			defer func() {
+				if err := recover(); err != nil {
+					logger.GetLogger().WithField("err", err).Warnf("RetryRunningProcByPid panic")
+				}
+			}()
```

Comment:
Is there any case where the call to `RetryRunningProcByPid` can panic?

Reply:
due to the use of goroutine, it is not expected that the process will exit due to unexpected panic